backend/src/services/smtp/smtp.ts

```diff
@@ -68,11 +72,10 @@ export async function requestReviewForAccessRequest(
   }
 
   const emailContent = buildEmail(
-    `Request for Entities '${accessRequest.metadata.overview.entities}' access to the model '${accessRequest.modelId}'`,
+    `${requestingEntitiesText(accessRequest.metadata.overview.entities.length)} requesting access to model ${accessRequest.modelId}`,
```

Comment:
I'm assuming you've done this because the raw list of entities isn't nice to look at. Normally I'd suggest using getUserInformation() so convert them all to friendly name but here we might have both users and groups. How do we present groups nicely in the frontend if we don't have a group equivalent for getUserInformation()?